app/ui/design-system/src/lib/Components/TabMenu/index.tsx

```diff
@@ -29,13 +40,14 @@ const TabMenu = ({ tabs, onTabChange, centered }: TabMenuProps) => {
         )
 
```

Comment:
```suggestion
// TODO: Return a button when onTabChange is provided, and an a tag for links. Make onTabChange optional
```

I split this out to #229